src/v8runtime/V8Inspector.h

```diff
@@ -9,8 +9,14 @@
 
 #include <cxxreact/MessageQueueThread.h>
 #include <condition_variable>
-#include "jsinspector/InspectorInterfaces.h"
 #include "v8-inspector.h"
+#ifdef REACT_NATIVE_0_73
+#include "jsinspector-modern/InspectorInterfaces.h"
+using IRemoteConnection = facebook::react::jsinspector_modern::IRemoteConnection;
+#else
+#include "jsinspector/InspectorInterfaces.h"
+using IRemoteConnection = facebook::react::IRemoteConnection;
+#endif
```

Comment:
maybe we can use class forward declaration and move these "includes" into V8Inspector.cpp

```cpp
namespace facebook {
namespace react {

class IRemoteConnection;

} // namespace react
} // namespace facebook
```